integration_tests/src/main/python/parquet_test.py

```diff
@@ -30,7 +30,7 @@ def read_parquet_sql(data_path):
 
 parquet_gens_list = [[byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
     string_gen, boolean_gen, date_gen,
-    TimestampGen(start=datetime(1590, 1, 1, tzinfo=timezone.utc))],
+    TimestampGen(start=datetime(1900, 1, 1, tzinfo=timezone.utc))],
```

Comment:
We don't support Parquet timestamps before 1900 under any conf settings?

Reply:
In practice `LEGACY` and `CORRECTED` in the UTC time zone between 1590 and 1900 are both OK, but I thought it best to match what spark was doing for `LEGACY` which is 1900 and above. If you want separate test ranges for each one I can do it, but I didn't think it was that valuable.

Reply:
@revans2 Can you point out where in Spark code the year `1900` date/time is checked? I only see the switch time at `1582-10-15`.

Reply:
I got it from some of the docs.

https://github.com/apache/spark/blob/261b281e6e57be32eb28bf4e50bea24ed22a9f21/docs/sql-error-conditions-inconsistent-behavior-cross-version-error-class.md?plain=1#L43-L45

They also have it in some tests.

https://github.com/apache/spark/blob/261b281e6e57be32eb28bf4e50bea24ed22a9f21/sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DateTimeRebaseBenchmark.scala#L66-L70

but honestly I don't know in the code where that came from. But I saw it in the docs so I decided it would be best to follow it because I didn't know where the issue was.